picr22-enne-1-voistlust/segment_module/test.py

```diff
@@ -0,0 +1,50 @@
+import segment
+import numpy as np
+import cPickle as pickle
+import time
+import cv2
+
+with open('../colors.pkl', 'rb') as fh:

```

Comment:
Are you using this file?

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it